spikeinterface/sortingcomponents/waveforms/pca.py

```diff
@@ -0,0 +1,175 @@
+import pickle
+from pathlib import Path
+
+import numpy as np
+from sklearn.decomposition import IncrementalPCA
+
+from spikeinterface.sortingcomponents.peak_pipeline import  PipelineNode
+from spikeinterface.sortingcomponents.peak_detection import detect_peaks
+from spikeinterface.sortingcomponents.peak_selection import select_peaks
+from spikeinterface.postprocessing import compute_principal_components
+from spikeinterface.core import BaseRecording
+from spikeinterface.core.sparsity import ChannelSparsity
+from spikeinterface import extract_waveforms, NumpySorting
+from spikeinterface.core.job_tools import _shared_job_kwargs_doc
+from spikeinterface.core.recording_tools import get_channel_distances
+from .waveform_utils import to_temporal_representation, from_temporal_representation
+
+
+class PCBaseNode(PipelineNode):
```

Comment:
```suggestion
class tPCABaseNode(PipelineNode):
```
Should we rename it to tPCA to keep the semantic of the Paninski group? (same for the file `tpca.py`)

Reply:
I think this is a good idea, I would go for the full Temporal thought.